src/main/java/com/raoulvdberge/refinedstorage/proxy/CapabilityNetworkNode.java

```diff
@@ -0,0 +1,62 @@
+package com.raoulvdberge.refinedstorage.proxy;
+
+import com.google.common.base.Preconditions;
+import com.raoulvdberge.refinedstorage.api.network.INetworkMaster;
+import com.raoulvdberge.refinedstorage.api.network.INetworkNode;
+import net.minecraft.item.ItemStack;
+import net.minecraft.nbt.NBTBase;
+import net.minecraft.nbt.NBTTagCompound;
+import net.minecraft.util.EnumFacing;
+import net.minecraftforge.common.capabilities.Capability;
+import net.minecraftforge.common.capabilities.CapabilityInject;
+import net.minecraftforge.common.capabilities.CapabilityManager;
+
+import javax.annotation.Nullable;
+
+public class CapabilityNetworkNode {
+    @CapabilityInject(INetworkNode.class)
+    public static Capability<INetworkNode> NETWORK_NODE_CAPABILITY = null;
```

Comment:
Having INetworkNode in the API should be enough, other mods will use `@CapabilityInject(INetworkNode.class)`. I'm trying a bit in enderio to ensure it will integrate as expected.